services/blockchain-app-registry/Dockerfile

```diff
@@ -1,24 +1,28 @@
+##### Stage 1
+
 FROM node:16-alpine AS builder
 
-RUN apk add --no-cache alpine-sdk python3 git
 RUN adduser -D builder
 
 USER builder
-RUN mkdir /home/builder/build
```

Comment:
This needs to be left in here or else the build fails (there won't be a build directory)
```
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /home/builder/build/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/home/builder/build/node_modules'
npm ERR!  [Error: EACCES: permission denied, mkdir '/home/builder/build/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/home/builder/build/node_modules'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/builder/.npm/_logs/2023-01-26T08_33_05_685Z-debug-0.log
```

Reply:
Same deal for the other Dockerfiles where this was removed

Reply:
Updated the Dockerfiles to get it working on Linux too. Images now get built on Linux, Macbook M1 and Macbook Intel.
Please re-verify.